assets/stylesheets/custom.scss

```diff
@@ -1,237 +1,201 @@
-				}
-			}
-
-			&.none {
-				background-color: #555;
-			}
-
-			&.rating {
-				border-radius: 5px;
-				background: darken($orange, 10%);
-				padding: 0px 4px;
-				color: black;
-				font-size: 10px;
-
-				&.rating-0 {
-					background-color: $red;
-				}
-
-				&.rating-1 {
-					background-color: $red;
-				}
-
-				&.rating-2 {
-					background-color: $red;
-				}
-
-				&.rating-3 {
-					background-color: darken($orange, 10%);
-				}
-
-				&.rating-4 {
-					background-color: darken($orange, 10%);
-				}
-
-				&.rating-5 {
-					background-color: darken($orange, 10%);
-				}
-
-				&.rating-6 {
-					background-color: darken($orange, 10%);
-				}
-
-				&.rating-7 {
-					background-color: darken($green, 10%);
-				}
-
-				&.rating-8 {
-					background-color: darken($green, 10%);
-				}
-
-				&.rating-9 {
-					background-color: darken($green, 10%);
-				}
-
-				&.rating-10 {
-					background-color: darken($green, 10%);
-				}
-			}
-
-			a {
-				padding: 2px 4px;
-			}
-
-			a, ul, li {
-				display: inline-block;
-			}
-
-			li {
-				font-size: 11px;
-				border-radius: 3px;
-				margin: 0 5px 0 0;
-				padding: 0;
-			}
-		}
-	}
-
-	&.issues_stacked, &.issues_opened, &.forum_unanswered {
-		// Reverse trends, up is bad
-		.trend-up {
-			color: $red;
-		}
-		.trend-down {
-			color: $green;
-		}
-	}
+  vertical-align: top;
+
+  &.widget-graph,
+  &.widget-series {
+    .more-info {
+      bottom: auto;
+      position: inherit;
+      top: auto;
+    }
+  }
+
+  &.widget-table {
+    td {
+      &.trend-up {
+        color: $green;
+      }
+
+      &.trend-down {
+        color: $red;
+      }
+    }
+
+  }
+
+  &.meta_stats {
+    background-color: transparent;
+    color: $grey;
+  }
+
+  &.travis,
+  &.testable_without_ci {
+    font-size: 14px;
+
+    ul {
+      margin: 0;
+    }
+
+    li {
+      border-radius: 3px;
+      float: left;
+      margin: 0 5px 5px 0;
+      padding: 3px;
+
+      // scss-lint:disable SelectorDepth
+      * {
+        vertical-align: middle;
+      }
+      // scss-lint:enable SelectorDepth
+
+      &.good {
+        background-color: darken($green, 10%);
+      }
+
+      &.bad {
+        background-color: $red;
+
+        // Show broken builds larger,
+        // with labels on a separate line.
+        // scss-lint:disable SelectorDepth, NestingDepth
+        .label {
+          display: block;
+          font-size: 24px;
+        }
+
+        ul {
+          display: block;
+
+          .label {
+            font-size: 14px;
+          }
+        }
+        // scss-lint:enable SelectorDepth, NestingDepth
+      }
+
+      // scss-lint:disable SelectorDepth
+      a {
+        padding: 2px 4px;
+      }
+
+      a,
+      ul,
+      li {
+        display: inline-block;
+      }
+
+      li {
+        border-radius: 3px;
+        font-size: 11px;
+        margin: 0 5px 0 0;
+        padding: 0;
+      }
+      // scss-lint:enable SelectorDepth
+    }
+  }
+
+  &.issues_stacked,
```

Comment:
Selector `issues_stacked` should be written in lowercase with hyphens<br>Selector `issues_opened` should be written in lowercase with hyphens<br>Selector `forum_unanswered` should be written in lowercase with hyphens